aws/ecs/variables.tf

```diff
@@ -215,13 +212,14 @@ variable "enable_container_insights" {
   nullable = false
 }
 
-variable "multi_az" {
-  type    = bool
-  default = false
+variable "alb_subnet_ids" {
+  type = list(string)
+  validation {
+    condition     = length(var.alb_subnet_ids) >= 2
```

Comment:
Do we really need more than one subnet? What if we want to deploy e.g. a staging environment to only a single AZ to safe cost?